### DISCUSSION TITLE
Template Studio still Targets .Net 7 + WinUI 3

### DISCUSSION THREAD

**Discussion by thinkgandhi** [microsoft/WindowsAppSDK#4929] [General]
Any plans to upgrade this Template Studio ?

**Comment by mklemarczyk** [comment]
Template Studio is managed via this repository, you can open an issue/discussion there.
https://github.com/microsoft/TemplateStudio/issues

I seen you posted already about update to .NET 9
https://github.com/microsoft/TemplateStudio/discussions/4769
